console/console-init/ui/src/modules/device/components/SecretList/SecretList.tsx

```diff
@@ -71,12 +71,17 @@ export const SecretList: React.FC<ISecretListProps> = ({
     handleInputChange(id, evt, value, secretId, "secrets");
   };
 
-  const defaultSecretsSetting = (
-    secret: ISecret,
-    id: string,
-    index: number
-  ) => {
+  const DefaultSecretsSetting: React.FC<{
+    secret: ISecret;
+    id: string;
+    index: number;
+  }> = ({ secret, id, index }) => {
     const isRequired = index === 0 ? true : false;
```

Comment:
Just the condition index===0 is enough to assign true or false

Reply:
It will fix in next PR.